src/common/module/expand_mod.F90

```diff
@@ -234,11 +234,18 @@ subroutine expand_i1(buffer, field, nlon, nproma, ngptot, nblocks)
   end subroutine expand_i1
 
   subroutine expand_r1(buffer, field, nlon, nproma, ngptot, nblocks)
+    use omp_lib
```

Comment:
@sbrdar no need to add "use omp_lib" unless you are calling actual routines instead of pragmas.